adhocracy4/maps/widgets.py

```diff
@@ -22,9 +22,9 @@ def render(self, name, value, attrs):
 
         context = {
             'map_url': settings.BASE_MAP,
-            'bbox': json.dumps(settings.MAP_BOUNDING_BOX),
```

Comment:
Also most other A4 settings are prefixed with A4. Should we change it for this one?